test/unittests/t_catalog_manager.cc

```diff
@@ -1,33 +0,0 @@
-/**
- * This file is part of the CernVM File System.
- */
-
-#include <gtest/gtest.h>
-
-#include <sys/stat.h>
-#include <unistd.h>
-
-#include "../../cvmfs/catalog.h"
-#include "../../cvmfs/catalog_mgr.h"
-#include "../../cvmfs/catalog_rw.h"
-#include "../../cvmfs/hash.h"
-#include "../../cvmfs/shortstring.h"
-#include "../../cvmfs/util.h"
-#include "testutil.h"
-
-using namespace std;  // NOLINT
-
-
-namespace catalog {
-
-
-
-
-
-
-
-
-
-
-
-}  // namespace catalog
```

Comment:
That was an error I introduced in the last PR. The actual file where the catalog manager tests are is t_catalog_mgr.cc, and not t_catalog_manager.cc